wrangler-service/src/main/java/io/cdap/wrangler/service/directive/AbstractDirectiveHandler.java

```diff
@@ -156,8 +158,18 @@ protected DirectiveExecutionResponse generateExecutionResponse(
     List<Map<String, Object>> values = new ArrayList<>(rows.size());
     Map<String, String> types = new HashMap<>();
     Set<String> headers = new LinkedHashSet<>();
```

Comment:
Can we merge `headers` and `types` into a single `LinkedHashMap`? In this function, `types` are generated from output schema and `headers` are generated from row data. Both should be generated from output schema. Also maintaining 2 different objects containing same information can lead to issues.

Reply:
Done, verified that LinkedHashMap's keyset method also maintains ordering